extensions/token/token-validation/src/main/java/org/eclipse/dataspaceconnector/token/JwtClaimValidationRule.java

```diff
@@ -14,8 +14,8 @@
 
 package org.eclipse.dataspaceconnector.token;
 
-import com.nimbusds.jwt.JWTClaimsSet;
+import com.nimbusds.jwt.SignedJWT;
 import org.eclipse.dataspaceconnector.spi.iam.ValidationRule;
 
-public interface JwtClaimValidationRule extends ValidationRule<JWTClaimsSet> {
+public interface JwtClaimValidationRule extends ValidationRule<SignedJWT> {
```

Comment:
Should the class be renamed to `JwtValidationRule`?

Reply:
Done (please check again if ok)